lib/Url/Profile.php

```diff
@@ -577,7 +577,7 @@ protected function buildQuery()
         }
         $query->whereRaw('('.implode(' OR ', $whereRawSegmentParts).')');
 
-        if ($this->hasRestrictions()) {
+        if ($this->hasRestrictions() || $this->getArticleClangId() > 0) {
```

Comment:
Wozu machst du das hier? Der Part hat mit Relationen nichts zu tun.

Reply:
Richtig, dieser Part und der in den Zeilen 596 hat mit den Relationen nichts zu tun. Dieser Code ist nur interessant, wenn im Profil im Feld "In welchem Artikel erfolgt die Ausgabe" in der Sprache nur eine Sprache ausgewählt ist. Zur Verdeutlichung habe ich in einem Profil die Sprache mit der ID 2 ausgewählt.

a) Hier das Ergebnis und die Query, die am Ende dieser Funktion herauskommt, wenn mein Code NICHT eingefügt ist (also Originalcode von dir):
![Bildschirmfoto von 2019-05-24 21-23-38](https://user-images.githubusercontent.com/4131072/58352022-a7e5d000-7e6a-11e9-8f66-bc2813a10322.png)

b) Hier das Ergebnis mit meiner Änderung:
![Bildschirmfoto von 2019-05-24 21-24-24](https://user-images.githubusercontent.com/4131072/58352085-ccda4300-7e6a-11e9-8613-74822dda0960.png)

Da im Profil nur eine Sprache ausgewählt ist, ist Ergebnis b) korrekt. Im Ergebnis a) sind Datensätze für 3 Sprachen zu viel enthalten.